src/zend_container.h

```diff
@@ -3,6 +3,34 @@
 
 extern "C" {
 #include "php.h"
+
+    //c func borrowed from https://github.com/php/php-src/blob/e08ce4c13db6e9aecd3497cd270b72d06c649bc7/ext/standard/array.c#L245
```

Comment:
Could you please add a space after `//` and format this file using clang-format? There is already a `.clang-format` in the project.

Reply:
done. force pushed